ami_configs/bld-linux64.json

```diff
@@ -1,30 +1,6 @@
 {
-    "us-west-1": {
-        "ami": "ami-fe393ebb",
-        "instance_type": "c3.xlarge",
-        "arch": "x86_64",
-        "distro": "centos",
-        "root_device_type": "instance-store",
```

Comment:
This config is not used in production. It was  added as a part of Bug 1085520 to support instance-store root device (no EBS, AMIs live in an S3 bucket).

For this type of golden images the underlying AMI doesn't really matter, because the process is a bit different. Golden AMIs for instance-store root device based images are created from scratch (without any base AMI) every time in a chroot environment which uses a file based image (mounted as a loop device). In other words changing the AMI here won't change the content  of the result AMI.